File: cross-platform-h5/android-h5-file-upload/app/src/main/java/com/chiclaim/webview/MainActivity.java
package com.chiclaim.webview;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);
    }


    public void appChooser(View view) {
        startActivity(new Intent(this, AppChooserActivity.class));
    }

    public void customChooser(View view) {
        startActivity(new Intent(this, CustomChooserActivity.class));
    }
}

File: android.architecture/src/main/java/com/chiclaim/android/arch/MainActivity.kt
package com.chiclaim.android.arch

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }


    fun testLifecycle(view: View) {
        startActivity(Intent(this, TestLifecycleActivity::class.java))
    }
}